### DISCUSSION TITLE
Interesting article about marimo from user of jupyter and panel

### DISCUSSION THREAD

**Discussion by misolietavec** [marimo-team/marimo#703] [General]
I am `jupyter(lab)` and `panel` (repository https://github.com/holoviz/panel) user too. There are (yet) not so many comparison between `marimo` and `panel`. I really liked this article

https://medium.com/@flyingjony/can-marimo-replace-jupyter-notebooks-fb8c7210ad35

I agree with all the author said. I can add, that `marimo` works well with `polars`  (`pandas` replacement) and `plotly`.

**Comment by akshayka** [comment]
Thank you for sharing this article. It is well-written and a good read, with some good feedback on how we can improve.
